lib/nextstrain/sphinx/theme/__init__.py

```diff
@@ -15,3 +15,6 @@
 
 def setup(app):
     app.add_html_theme('nextstrain-sphinx-theme', str(package_dir))
+    app.setup_extension('sphinx_copybutton')
+    # customize sphinx_copybutton https://sphinx-copybutton.readthedocs.io/en/latest/use.html
+    app.config['copybutton_prompt_text'] = '$ '
```

Comment:
Would also be good to configure the [line continuation char](https://sphinx-copybutton.readthedocs.io/en/latest/use.html#honor-line-continuation-characters-when-copying-multline-snippets), I think, as that's used in some of our examples.

Does the above `app.config[k] = v` assignment preclude project-specific assignments to those options? If so, probably better to use `app.config.setdefault(k, v)` instead (assuming `app.config` is a normal dict or otherwise provides the full mapping protocol).

Reply:
> Does the above `app.config[k] = v` assignment preclude project-specific assignments to those options?

No, this seems to be the order of operations on `config`:

1. In this `__init__.py`, `app.setup_extension('sphinx_copybutton')` sets the config value to `''`.
2. In this `__init__.py`, `app.config['copybutton_prompt_text'] = '$ '` sets the default for the `nextstrain.sphinx.theme` extension.
3. In a `conf.py`, `copybutton_prompt_text = '% '` overrides any previously set value.

Either way, I looked at using `setdefault` but `app.config` is a [`sphinx.config.Config`](https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.config.Config) which doesn't implement `dict` nor `setdefault` so that wouldn't be an option.

Reply:
Added line continuation char: 3300943